components/Footer.tsx

```diff
@@ -0,0 +1,21 @@
+import { Copyright } from "lucide-react";
+
+const Footer = () => {
+  return (
+    <footer className="fixed bottom-0 w-full h-20 flex justify-between items-center bg-white/10 p-4 lg:px-14 border border-t-gray-200/90">
+      <div className="flex justify-center items-center gap-4 font-normal text-slate-600">
+        <p>LinkedIn</p>
```

Comment:
add logos instead of names

Reply:
added the function to get repo stars, will update the badge soon with better design reference

Reply:
added the logos, but some of the logo are been deperecated by lucide-react, will replace them with svg